### DISCUSSION TITLE
社区支持 | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#467] [General]
# 社区支持 | FixIt

Hugo FixIt 是一个社区驱动的项目。我们欢迎所有的反馈和贡献。

https://fixit.lruihao.cn/zh-cn/community/

**Comment by Lruihao** [comment]
这个评论区也可以用来集中回复和讨论一些小问题～:P

**Reply by Lruihao** [reply]
当然也可以是吹水区🫧

**Reply by Lruihao** [reply]
互动起来朋友们，盖楼盖楼🏢
